sample-kotlin/src/main/kotlin/social/bigbone/sample/ManageFilters.kt

```diff
@@ -85,7 +88,10 @@ object ManageFilters {
      * @param client a [MastodonClient] with an authenticated user
      * @param filterId ID string for the filter that should be deleted
      */
-    private fun deleteFilter(client: MastodonClient, filterId: String) {
+    private fun deleteFilter(
+        client: MastodonClient,
+        filterId: String
```

Comment:
What’s the rule here? We use separate lines for methods with more than one parameter?

Reply:
Yes, that's the ktlint standard rule for parameters.